src/test/java/guitests/InvalidLoginTest.java

```diff
@@ -32,9 +31,8 @@ public void invalidLoginTest() throws InterruptedException {
 
     @Test
     public void emptyLoginTest() throws InterruptedException {
-        TextField repoOwnerField = find("#repoOwnerField");
-        doubleClick(repoOwnerField);
-        doubleClick(repoOwnerField);
+        find("#repoOwnerField");
+        selectAll();
```

Comment:
An abstraction that could be nice is to wait until a node appears, find, click, select all, then type a string. Seems like this is frequently done but I don't know exactly how much.